src/hooks/useWindowDimensions.js

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";

```

Comment:
custom react hook to get the window size. This is only necessary for more complex responsiveness. For example, the photo and text should alternate position on large screens, however they shouldn't alternate position on small screens because there will only be one column.